sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/MediaStreamingPackageParserUnitTests.java

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation;
+
+import com.azure.communication.callautomation.models.MediaStreamingAudio;
+import com.azure.communication.callautomation.models.MediaStreamingMetadata;
+import com.azure.communication.callautomation.models.MediaStreamingPackageParser;
+import com.azure.core.util.BinaryData;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.jupiter.api.Test;
+import java.nio.charset.StandardCharsets;
+import java.time.OffsetDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class MediaStreamingPackageParserUnitTests {
+    @Test
+    public void parseAudioData() {
+        String audioJson = "{"
+            + "\"kind\": \"AudioData\","
```

Comment:
I'm sorry, I should have checked this before. 
This data is set is wrong, this is the right contract:
```
{
    "kind": "AudioData",
    "timestamp": "2022-09-15T01:30:35.356Z",
    "participantRawId": "abc123",
    "data": "Bg==", // to be documented as base64-encoded string; not byte[]
    "silent": false
}
```

Everything is flatten. I don't know if this has implication in your code, but you could check it out.
Reference: https://skype.visualstudio.com/SPOOL/_workitems/edit/2968625